integrated_channels/integrated_channel/models.py

```diff
@@ -78,6 +78,12 @@ class EnterpriseCustomerPluginConfiguration(TimeStampedModel):
             "selected from the newest catalog."
         ),
     )
+    disable_learner_data_transmissions = models.BooleanField(
+        default=False,
+        verbose_name="Learner Data Transmission Feature Gate",
```

Comment:
will this be better named like a verb similar to our other booleans? such as 

```
Disable Learner Data Transmission
```

e.g., we have things like `Enable audit data reporting` , `Hide course original price`

so just thinking it will look a little bit more consistent

Reply:
consistent and I think it's clearer to the user who's toggling this in Django Admin